### ISSUE TITLE
Vague reference to obfuscation in docs

### ISSUE THREAD

**Issue by Tbhesswebber** [flutter/website#4210]
Page URL: https://flutter.dev/docs/deployment/obfuscate.html
Page source: https://github.com/flutter/website/tree/master/src/docs/deployment/obfuscate.md

The Obfuscating Dart Code page references that we are able to perform obfuscation, but never mentions what types of obfuscation are deployed.  Is this merely name obfuscation?  Is it obfuscation at the Dart level before compilation or is this obfuscation of the compiled assembly code using overlapping assembly instructions or some other technique?

If it's name obfuscation, are there tools available to add further layers of obfuscation to the application?

**Comment by sfshaza2** [comment]
@jonahwilliams, can you elaborate? (or delegate)

**Comment by jonahwilliams** [comment]
The command line option contains sufficient documentation:

```
In a release build, this flag removes identifiers and replaces them with randomized values for the purposes of source code obfuscation. This flag must
                                                  always be combined with "--split-debug-info" option, the mapping between the values and the original identifiers is stored in the symbol map created in
                                                  the specified directory. For an app built with this flag, the 'flutter symbolize' command with the right program symbol file is required to obtain a human
                                                  readable stack trace.
```

TLDR: identifiers are renamed, that is it

**Comment by Tbhesswebber** [comment]
The phrase "source-code obfuscation" is what feels vague - replacing it with the phrase "name obfuscation within the source-code" is more specific _and_ easily Google-able for those who don't quite understand the security implications of specific types of obfuscation.

Is there a reason that only name obfuscation is utilized? Are there mitigating factors within Flutter and/or Dart that make other layers of obfuscation less valuable or is this, understandably, an issue of complexity versus ROI?

**Comment by MiraMarshall** [comment]
This issue has been addressed. The language vague language is no longer on the page.
